types/knockout/index.d.ts

```diff
@@ -656,7 +782,13 @@ interface KnockoutStatic {
     renderTemplateForEach(template: Function, arrayOrObservableArray: KnockoutObservable<any>, options: Object, targetNode: Node, parentBindingContext: KnockoutBindingContext): any;
     renderTemplateForEach(template: any, arrayOrObservableArray: KnockoutObservable<any>, options: Object, targetNode: Node, parentBindingContext: KnockoutBindingContext): any;
 
-    ignoreDependencies<T>(callback: () => T): T;
+    /**
+     * Executes a callback function inside a computed observable, without creating a dependecy between it and the observables inside the function
+     * @param callback Function to be called.
+     * @param callbackTarget? Defines the value of 'this' in the callback function
+     * @param callbackArgs? Arguments for the callback Function
+     */
+    ignoreDependencies<T>(callback: () => T, callbackTarget?: any, callbackArgs?: any): T;
```

Comment:
This could be improved to make the callback aware of the target and the arguments:

```ts
ignoreDependencies<T>(callback: () => T): T;
ignoreDependencies<T, Target, Args extends any[]>(
    callback: (this: Target, ...args: Args) => T, callbackTarget?: Target, ...callbackArgs: Args,
): T;
```

But maybe that's best addressed in a different PR, as there's a number of places this pattern could be used (and the args bit would require bumping the minimum version of TS for this library to 3.0)